prov/util/src/util_attr.c

```diff
@@ -630,6 +630,7 @@ int ofi_check_ep_attr(const struct util_prov *util_prov, uint32_t api_version,
 	const struct fi_provider *prov = util_prov->prov;
 
 	if ((user_attr->type != FI_EP_UNSPEC) &&
+	    (prov_attr->type != FI_EP_UNSPEC) &&
```

Comment:
Why would a provider set this to UNSPEC?

Reply:
The multi-rail provider can set this to UNSPEC so that apps can use it run over RDM as well as DGRAM underlying provider.